Lib/test/test_embed.py

```diff
@@ -32,7 +32,7 @@
 def debug_build(program):
     program = os.path.basename(program)
     name = os.path.splitext(program)[0]
-    return name.endswith("_d")
+    return name.casefold().endswith("_d".casefold())
```

Comment:
Because `casefold` is the correct method for case-insensitive comparisons. `lower` is the correct method where you want it to be lowercase.

ASCII doesn't have any difference, but Unicode does.

Reply:
Then again, it's a well known string, so `.lower` would have been fine :D Just out of habit, I guess

Reply:
Oh, I didn't ask to change the code, I was just curious :-)